src/posthog-featureflags.ts

```diff
@@ -433,7 +433,7 @@ export class PostHogFeatureFlags {
 
         if (!existing_early_access_features || force_reload) {
             this.instance._send_request({
-                transport: 'XHR',
+                transport: 'fetch',
```

Comment:
can this just not specify their own transport and use the default instead?

Reply:
Probably.

Reply:
https://github.com/PostHog/posthog-js/pull/1614